block/retriever.go

```diff
@@ -79,6 +79,7 @@ func (m *Manager) processNextDABatch(daMetaData *da.DASubmitMetaData) error {
 	m.logger.Debug("trying to retrieve batch from DA", "daHeight", daMetaData.Height)
 	batchResp := m.fetchBatch(daMetaData)
 	if batchResp.Code != da.StatusSuccess {
+		m.logger.Error("Failed fetching batch from DA", batchResp.Message)
```

Comment:
@danwt wouldn't like that u use the "Failed' word in an error logger :) Better remove.

Reply:
msg udpated